detect_secrets_server/storage/base.py

```diff
@@ -134,6 +137,9 @@ def _construct_debugging_output(self, sha):  # pragma: no cover
     def get_last_commit_hash(self):
         return git.get_last_commit_hash(self._repo_location)
 
+    def get_empty_repo_commit_hash(self):
+        return git.get_empty_tree_commit_hash()
```

Comment:
We don't need to expose `git` through the storage abstraction, especially if it's just a direct proxy. Instead, can just do:

```python
from detect_secrets_server.storage.core import git
```

in `detect_secrets_server/repos/base_tracked_repo.py`.

Reply:
++, good catch!